src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue18933.cs

```diff
@@ -0,0 +1,23 @@
+using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+
+namespace Microsoft.Maui.TestCases.Tests.Issues;
+
+public class Issue18933 : _IssuesUITest
+{
+	public Issue18933(TestDevice device) : base(device)
+	{
+	}
+
+	public override string Issue => "ContentView Background Color Not Cleared When Set to Null";
+
+	[Test]
+	[Category(UITestCategories.ViewBaseTests)]
+	public void VerifyBackgroundColorCleared()
+	{
+		App.WaitForElement("clearBgBtn");
+		App.Tap("clearBgBtn");
+		VerifyScreenshot();
```

Comment:
Pending snapshot on Mac and Windows.
<img width="789" height="563" alt="image" src="https://github.com/user-attachments/assets/78bbcdd7-e407-4e13-bc23-fc261cda6bbe" />
Could you commit the images?